ludwig/utils/trainer_utils.py

```diff
@@ -179,6 +180,7 @@ def __init__(
         self.train_metrics = train_metrics
         self.validation_metrics = validation_metrics
         self.test_metrics = test_metrics
+        self.llm_eval_examples = llm_eval_examples
```

Comment:
Can you add a comment here with an example of the expected structure and how this is only set for LLM fine-tuning?

> llm_eval_examples is used to capture samples of prompts/outputs for LLM fine-tuning. There are 3 keys (inputs, targets, outputs), which each map to a list of strings.